instrumentation/p6spy/src/test/java/brave/p6spy/TracingJdbcEventListenerTest.java

```diff
@@ -121,6 +128,34 @@
     verify(span).remoteIpAndPort("1.2.3.4", 5555);
   }
 
+  @Test
+  @TestCaseName("remoteServiceName should be '{1}'")
+  @Parameters({
```

Comment:
this dep hasn't been updated since 2017, so might haunt us later. you mind switching to just using a map and calling a method not marked test for each pair here?

Reply:
unfortunately for junit4 there are no better alternatives, junit5 has support for this style of tests out of the box, let me see what I can do.

Reply:
Arrays.asList in a for loop is better than a dead dependency. can you do add this? I'm worried about best being enemy of good..